timemachine/cpp/src/mol_utils.cpp

```diff
@@ -1,6 +1,7 @@
 #include "mol_utils.hpp"
 #include <algorithm>
 #include <set>
+#include <stdexcept>
```

Comment:
is this used?

Reply:
This is used for the runtime error. Added since under newer versions of GCC this doesn't compile without the addition. Can remove, but seemed like a harmless addition to compile with newer GCC

Reply:
No need to remove, makes sense.